internal/characters/dori/burst.go

```diff
@@ -0,0 +1,101 @@
+package dori
+
+import (
+	"math"
+
+	"github.com/genshinsim/gcsim/internal/frames"
+	"github.com/genshinsim/gcsim/pkg/avatar"
+	"github.com/genshinsim/gcsim/pkg/core/action"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/event"
+	"github.com/genshinsim/gcsim/pkg/core/player"
+)
+
+var burstFrames []int
+
+const (
+	burstHitmark    = 28
+	burstHealPeriod = 12 * 60 / 6
```

Comment:
don't think this calc and variable is needed, because it's just every 120 frames